src/main/java/seedu/clinic/logic/commands/HelpCommand.java

```diff
@@ -118,7 +125,10 @@ private String generateHelpGenericMessage() {
         String aboutUpdateCommand = UpdateCommand.COMMAND_WORD + "\nUpdates a product associated with a supplier or"
                 + " warehouse";
         String aboutViewCommand = ViewCommand.COMMAND_WORD + "\nViews the in-depth information associated with a"
-                + "specific supplier or warehouse";
+                + " specific supplier or warehouse";
+        String aboutUndoCommand = UndoCommand.COMMAND_WORD + "\nUndo the previous editing on the data of CLI-nic";
+        String aboutRedoCommand = RedoCommand.COMMAND_WORD + "\nRedo the previously undone editing"
```

Comment:
This sounds a bit strange. Perhaps "Recovers the previous state of CLI-nic before undo command" would be clearer.

Reply:
Thanks!